lib/main.dart

```diff
@@ -148,7 +148,7 @@ class VisualEditorState extends State<VisualEditor>
   final _editorKey = GlobalKey<State<VisualEditor>>();
   final _editorRendererKey = GlobalKey<State<VisualEditor>>();
   KeyboardVisibilityController? keyboardVisibilityCtrl;
-  late final StreamSubscription<bool>? keyboardVisibilitySub;
```

Comment:
Hi @Phil9l! Sorry for the late comment. It has been super difficult to get time to review the PRs. Can you confirm? is this change required on the latest commit? 

Also I'd like to know, what happens? I haven't seen the error so far.

Reply:
Huh, looks like I provided an answer a while ago. Though it's not clear from the comment what was going on. For sure I know that I haven't seen the error in a long while so I'll close the PR.